src/egon/data/datasets/sanity_checks.py

```diff
@@ -0,0 +1,460 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jan 21 10:49:43 2022
+
+@author: student
+"""
```

Comment:
Please replace line 1 - 7 by a short description what the script is doing. See this example: https://github.com/openego/eGon-data/blob/dev/src/egon/data/datasets/vg250/__init__.py#L1-L10